core/deno.ts

```diff
@@ -7,7 +7,7 @@ export function getDenoDir(): string {
   // On Windows: %LOCALAPPDATA%/deno (%LOCALAPPDATA% = FOLDERID_LocalAppData)
   // On macOS: $HOME/Library/Caches/deno
   // If something fails, it falls back to $HOME/.deno
-  let denoDir = process.env.DENO_DIR;
+  let denoDir = process.env.DENO_DIR || process.env.VSCODE_CUSTOM_DENO_DIR;
```

Comment:
`process.env.VSCODE_CUSTOM_DENO_DIR` should be a higher priority

```ts
let denoDir = process.env.VSCODE_CUSTOM_DENO_DIR || process.env.DENO_DIR;
```

Reply:
Ok, I have updated both `DENO_DIR` and executable path settings to have a higher priority.